src/phoenix/server/bearer_auth.py

```diff
@@ -81,3 +84,16 @@ async def intercept(
                     return await method(request_or_iterator, context)
                 raise Unauthenticated()
         raise Unauthenticated()
+
+
+async def check_authenticated(request: Request) -> None:
```

Comment:
```suggestion
async def is_authenticated(request: Request) -> None:
```
might read better? non-blocking